accessibility/tools.md

```diff
@@ -0,0 +1,93 @@
+# Accessibility tools
+
+- [Automated testing](#automated-testing)
+  - [Pa11y and axe](#pa11y-and-axe)
+- [Manual testing](#manual-testing)
+- [Assistive technology](#assistive-technology)
+  - [Screen readers](#screen-readers)
+- [Other accessibility tools](#other-accessibility-tools)
+  - [Accessibility testing tools in the browser](accessibility-testing-tools-in-the-browser)
+  - [Tools for manually analysing web pages](tools-for-manually-analysing-web-pages)
```

Comment:
the links don't work as they are missing the hashes, should be:
- `#accessibility-testing-tools-in-the-browser`
- `#tools-for-manually-analysing-web-pages`